src/components/atoms/Article.vue

```diff
@@ -15,7 +15,7 @@
 	.article {
 		margin: 0;
 		padding: 0;
-		font-size: 12px;
+		font-size: rem(12px);
```

Comment:
why font size rem and letter spacing without rem?

Reply:
Just didn't seem logical to specify it in rem